core/src/solvers/cartesian_path.cpp

```diff
@@ -80,8 +80,8 @@ bool CartesianPath::plan(const planning_scene::PlanningSceneConstPtr& from, cons
 	kinematic_constraints::KinematicConstraintSet kcs(sandbox_scene->getRobotModel());
 	kcs.add(path_constraints, sandbox_scene->getTransforms());
 
-	auto isValid = [&sandbox_scene, &kcs](moveit::core::RobotState* state, const moveit::core::JointModelGroup* jmg,
-	                                      const double* joint_positions) {
+	auto is_valid = [&sandbox_scene, &kcs](moveit::core::RobotState* state, const moveit::core::JointModelGroup* jmg,
```

Comment:
Ah, clang-tidy considers is_valid as a (function) _variable_ here and thus enforces a lower_case name. 
I would still consider it a function. Sigh.

Reply:
The borders are quite fuzzy around that these days. :)
Technically it *is* a variable, although `auto` for closures creates a unique type signature for them...